### PR TITLE
Updated examples for Besu, Geth & Quorum

### PR DESCRIPTION
**Pull request by j4ys0n**
Besu:
This adds an example app to the existing Besu example which is intended to show how to link a custom app (and dashboards) with docker-compose and should help users with self-service.

Geth:
This adds two examples for Geth. One is a single-node dev example and the other is a multi-node private network example. Both have ethlogger plugged in and include an example app to assist users with self-service.

Quorum:
The Quorum example was removed in favor of using the `quorum-wizard` to generate a config & supporting resources. This is actually still [pending merge](https://github.com/jpmorganchase/quorum-wizard/pull/93) & release by the Quorum team.

### CODE REVIEW

File: examples/besu/docker-compose.yaml [hunk 1]
```diff
@@ -20,6 +20,7 @@ services:
             - ./splunk.yml:/tmp/defaults/default.yml
             - /opt/splunk/var
             - /opt/splunk/etc
+            - ./example-app:/opt/splunk/etc/apps/example
```
Comment (ziegfried) [inline examples/besu/docker-compose.yaml:23]: Just to follow up from the previous PR: what's this example app for? It doesn't seem to contain much, so can't really see the purpose.

File: examples/geth-single-node/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  a. single node (dev mode)
+  ```sh-session
+  $ cd examples/geth
+  $ docker-compose -f docker-compose-single.yaml up -d
+  ```
+
+  or
+
+  b. multi node (private network)
```
Comment (ziegfried) [inline examples/geth-single-node/README.md:19]: Now that the examples are split into separate directories, we'll probably need to update the instructions here.

**Comment by ziegfried** [comment]
You'll also need to fix code formatting, see https://github.com/splunk/splunk-connect-for-ethereum/blob/master/docs/developing.md#formatting

File: examples/besu/docker-compose.yaml (hunk 1 above)
Comment (j4ys0n) [reply examples/besu/docker-compose.yaml:23]: it's just to help users understand how to make an app

File: examples/geth-single-node/README.md (hunk 2 above)
Comment (j4ys0n) [reply examples/geth-single-node/README.md]: good catch, i forgot to change that. fixed now.

File: examples/besu/docker-compose.yaml (hunk 1 above)
Comment (ziegfried) [reply examples/besu/docker-compose.yaml:23]: I'm not sure it does help that much and is maybe rather confusing than helpful.

Comment (j4ys0n) [reply examples/besu/docker-compose.yaml:23]: no problem, it's removed.

File: examples/geth-single-node/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Ethlogger configuration is provided in the form of [environment variables](../../docs/cli.md#environment-variables) in [docker-compose.yaml](./docker-compose.yaml#L25).
+
+## Run
+
+1. Start docker-compose
+
+```sh-session
+$ cd examples/geth
+$ docker-compose -f docker-compose-single.yaml up -d
```
Comment (ziegfried) [inline examples/geth-single-node/README.md:13]: Should this be `docker-compose.yaml` now?

File: examples/besu/docker-compose.yaml (hunk 1 above)
Comment (ziegfried) [reply examples/besu/docker-compose.yaml:23]: This volume reference should be removed as well, right?

File: examples/geth-single-node/README.md (hunk 3 above)
Comment (j4ys0n) [reply examples/geth-single-node/README.md]: yep. good catch. thanks. updated.

File: examples/besu/docker-compose.yaml (hunk 1 above)
Comment (j4ys0n) [reply examples/besu/docker-compose.yaml]: done

**Pull request merged**